### PR TITLE
rds_instance: split up the integration tests

### PR DESCRIPTION
**Pull request by goneri**
Split the `rds_instance` target in smaller targets.

### CODE REVIEW

File: tests/integration/targets/rds_instance_modify/tasks/main.yml [hunk 1]
```diff
@@ -1,5 +1,13 @@
-#TODO: test availability_zone and multi_az
-- block:
+- name: rds_instance / modify integration tests
+  collections:
+  - community.aws
```
Comment (alinabuzachis) [inline tests/integration/targets/rds_instance_modify/tasks/main.yml:3]: I guess we can remove community.aws.

File: tests/integration/targets/rds_instance_processor/tasks/main.yml [hunk 2]
```diff
@@ -1,4 +1,13 @@
-- block:
+- name: rds_instance / processor integration tests
+  collections:
+  - community.aws
```
Comment (alinabuzachis) [inline tests/integration/targets/rds_instance_processor/tasks/main.yml:3]: Same here.

File: tests/integration/targets/rds_instance_replica/tasks/main.yml [hunk 3]
```diff
@@ -1,4 +1,13 @@
-- block:
+- name: rds_instance / replica integration tests
+  collections:
+  - community.aws
```
Comment (alinabuzachis) [inline tests/integration/targets/rds_instance_replica/tasks/main.yml:3]: Same here.

File: tests/integration/targets/rds_instance_restore/tasks/main.yml [hunk 4]
```diff
@@ -1,4 +1,13 @@
-- block:
+- name: rds_instance / restore integration tests
+  collections:
+  - community.aws
```
Comment (alinabuzachis) [inline tests/integration/targets/rds_instance_restore/tasks/main.yml:3]: Same here.

File: tests/integration/targets/rds_instance_sgroups/aliases [hunk 5]
```diff
@@ -0,0 +1,4 @@
+cloud/aws
+rds_instance_info
```
Comment (alinabuzachis) [inline tests/integration/targets/rds_instance_sgroups/aliases:2]: `rds_instance_info` doesn't seem to me to be used in this role.

File: tests/integration/targets/rds_instance_snapshot_mgmt/tasks/main.yml [hunk 6]
```diff
@@ -1,4 +1,13 @@
-- block:
+- name: rds_instance / snapshot_mgmt integration tests
+  collections:
+  - community.aws
```
Comment (alinabuzachis) [inline tests/integration/targets/rds_instance_snapshot_mgmt/tasks/main.yml:3]: Same here.

File: tests/integration/targets/rds_instance_states/tasks/main.yml [hunk 7]
```diff
@@ -1,4 +1,13 @@
-- block:
+- name: rds_instance / states integration tests
+  collections:
+  - community.aws
```
Comment (alinabuzachis) [inline tests/integration/targets/rds_instance_states/tasks/main.yml:3]: Same here.

File: tests/integration/targets/rds_instance_tagging/tasks/main.yml [hunk 8]
```diff
@@ -1,4 +1,13 @@
-- block:
+- name: rds_instance / tagging integration tests
+  collections:
+  - community.aws
```
Comment (alinabuzachis) [inline tests/integration/targets/rds_instance_tagging/tasks/main.yml:3]: Same here.

File: tests/integration/targets/rds_instance_upgrade/aliases [hunk 9]
```diff
@@ -0,0 +1,4 @@
+cloud/aws
+rds_instance_info
```
Comment (alinabuzachis) [inline tests/integration/targets/rds_instance_upgrade/aliases:2]: `rds_instance_info` doesn't seem to me to be used here.

File: tests/integration/targets/rds_instance_aurora/tasks/main.yml [hunk 10]
```diff
@@ -1,4 +1,13 @@
-- block:
+- name: rds_instance / aurora integration tests
+  collections:
+  - community.aws
```
Comment (alinabuzachis) [inline tests/integration/targets/rds_instance_aurora/tasks/main.yml:3]: I guess we can remove community.aws.

File: tests/integration/targets/rds_instance_aurora/aliases [hunk 11]
```diff
@@ -0,0 +1,6 @@
+# See: https://github.com/ansible-collections/amazon.aws/pull/1192
+disabled
+cloud/aws
+rds_instance_info
```
Comment (alinabuzachis) [inline tests/integration/targets/rds_instance_aurora/aliases:4]: `rds_instance_info` doesn't seem to me be used in this role.

File: tests/integration/targets/rds_instance_complex/aliases [hunk 12]
```diff
@@ -1,4 +1,4 @@
-slow
-
 cloud/aws
 rds_instance_info
```
Comment (alinabuzachis) [inline tests/integration/targets/rds_instance_complex/aliases:2]: Same here.

File: tests/integration/targets/rds_instance_modify/aliases [hunk 13]
```diff
@@ -0,0 +1,4 @@
+cloud/aws
+rds_instance_info
```
Comment (alinabuzachis) [inline tests/integration/targets/rds_instance_modify/aliases:2]: Same here.

File: tests/integration/targets/rds_instance_processor/aliases [hunk 14]
```diff
@@ -0,0 +1,4 @@
+cloud/aws
+rds_instance_info
```
Comment (alinabuzachis) [inline tests/integration/targets/rds_instance_processor/aliases:2]: Same here.

File: tests/integration/targets/rds_instance_aurora/aliases (hunk 11 above)
Comment (goneri) [reply tests/integration/targets/rds_instance_aurora/aliases:4]: Nice catch, it was also the case of rds_instance_complex, reds_instance_modify, rds_instance_processor, rds_instance_replica, rds_instance_restore rds_instance_sgroups, rds_instance_tagging, rds_instance_upgrade

**Review by alinabuzachis — APPROVED** [review]

**Pull request merged**